### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by poornas**
## Description

Some mc commands have changed in mc master (commit cb4f46854bd886ac60c74f42757e5e108d7c3733). References to these mc commands needs to be updated when a new version of mc is released. 
## Motivation and Context

Makes docs consistent
## Regression
 No

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by deekoder** [comment]
need this for release today.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#00c06beb-e3d4-495a-8576-d48425bf3598) |
<a name="00c06beb-e3d4-495a-8576-d48425bf3598"></a>
### 6893-8591ffa/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30973
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d5270ffbea4e:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 2 minutes and 35 seconds
{
  "name": "awscli",
  "duration": 73178,
  "function": "delete_bucket\n",
  "status": "FAIL",
  "error": "remove_bucket failed: s3://awscli-mint-test-bucket-12503 An error occurred (BucketNotEmpty) when calling the DeleteBucket operation: The bucket you tried to delete is not empty"
}

Executed 0 out of 13 tests successfully.
```

File: docs/zh_CN/bucket/notifications/README.md [hunk 1]
```diff
@@ -916,13 +916,13 @@ endpoint是监听webhook通知的服务。保存配置文件并重启Minio服务
...
 ```
 
 验证事件通知是否配置正确：
 
 ```
-mc events list myminio/images
+mc event list myminio/images
 ```
 
 你应该可以收到如下的响应：
```
Comment (eco-minio) [inline docs/zh_CN/bucket/notifications/README.md:928]: Does this section belong here or is this for the CN docs?

**Review by eco-minio — CHANGES_REQUESTED** [review]

File: docs/zh_CN/bucket/notifications/README.md (hunk 1 above)
Comment (poornas) [reply docs/zh_CN/bucket/notifications/README.md:928]: @eco-minio, this is the chinese version of the README

**Review by eco-minio — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**